sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/README.md

```diff
@@ -1,34 +1,68 @@
 # Azure Monitor Live Metrics client library for .NET
 
-TODO
+The [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet) compatible processor which sends [telemetry data](https://docs.microsoft.com/azure/azure-monitor/app/data-model) to [Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/app/app-insights-overview) [Live Metrics](https://learn.microsoft.com/azure/azure-monitor/app/live-stream).
 
 ## Getting started
 
+### Prerequisites
+
+- **Azure Subscription:**  To use Azure services, including Azure Monitor Exporter for [OpenTelemetry .NET](https://github.com/open-telemetry/opentelemetry-dotnet), you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://azure.microsoft.com/account).
+- **Azure Application Insights Connection String:** To send telemetry data to the monitoring service you'll need connection string from Azure Application Insights. If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [Create an Application Insights resource](https://docs.microsoft.com/azure/azure-monitor/app/create-new-resource) and [copy the connection string](https://docs.microsoft.com/azure/azure-monitor/app/sdk-connection-string?tabs=net#find-your-connection-string).
+
 ### Install the package
 
+#### Latest Version: [![Nuget](https://img.shields.io/nuget/vpre/Azure.Monitor.OpenTelemetry.LiveMetrics.svg)](https://www.nuget.org/packages/Azure.Monitor.OpenTelemetry.LiveMetrics/)  
```

Comment:
Will CI allow it to pass?

Reply:
I was wondering that too. Nope, it just failed, will remove.